shared/js/ui/templates/tracker-networks.es6.js

```diff
@@ -64,7 +64,7 @@ function renderTrackerDetails (model, DOMAIN_MAPPINGS) {
         </span>
     </div>
     <h1 class="site-info__domain block">${c.name}</h1>
-    <ol class="default-list site-info__trackers__company-list__url-list">
+    <ol class="default-list site-info__trackers__company-list__url-list" aria-label="Tracker domains for ${c.name}">
```

Comment:
@jagtalon I know you recommended using `dl` instead of `ol` but that required restructuring stuff quite a bit. I'm not sure if this view is going to be there long anyway.